packages/nuxeo-platform-explorer-package/src/main/resources/install/templates/explorer-deploy/nuxeo.prod

```diff
@@ -0,0 +1,153 @@
+
+# optional
+nuxeo.s3storage.cacheminage=${env:NUXEO_S3STORAGE_CACHEMINAGE:60}
+nuxeo.s3storage.cachesize=${env:NUXEO_S3STORAGE_CACHESIZE:100MB}
+nuxeo.s3storage.cachecount=${env:NUXEO_S3STORAGE_CACHECOUNT:10000}
+nuxeo.s3storage.connection.max=${env:NUXEO_S3STORAGE_CONNECTION_MAX:50}
+nuxeo.s3storage.connection.retry=${env:NUXEO_S3STORAGE_CONNECTION_RETRY:3}
+nuxeo.s3storage.connection.timeout=${env:NUXEO_S3STORAGE_CONNECTION_TIMEOUT:50000}
+nuxeo.s3storage.socket.timeout=${env:NUXEO_S3STORAGE_SOCKET_TIMEOUT:50000}
+nuxeo.s3storage.endpoint=${env:NUXEO_S3STORAGE_ENDPOINT:}
+nuxeo.s3storage.pathstyleaccess=${env:NUXEO_S3STORAGE_PATHSTYLEACCESS:false}
+nuxeo.s3storage.accelerateMode=${env:NUXEO_S3STORAGE_ACCELERATEMODE:false}
+nuxeo.s3storage.bucket_prefix=${env:NUXEO_S3STORAGE_BUCKET_PREFIX:}
+nuxeo.s3storage.crypt.keystore.file=${env:NUXEO_S3STORAGE_CRYPT_KEYSTORE_FILE:}
+nuxeo.s3storage.crypt.keystore.password=${env:NUXEO_S3STORAGE_CRYPT_KEYSTORE_PASSWORD:}
+nuxeo.s3storage.crypt.key.alias=${env:NUXEO_S3STORAGE_CRYPT_KEY_ALIAS:}
+nuxeo.s3storage.crypt.key.password=${env:NUXEO_S3STORAGE_CRYPT_KEY_PASSWORD:}
+nuxeo.s3storage.crypt.serverside=${env:NUXEO_S3STORAGE_CRYPT_SERVERSIDE:false}
+
+nuxeo.s3storage.useDirectUpload=${env:NUXEO_S3STORAGE_USEDIRECTUPLOAD:false}
+
+# mandatory
+nuxeo.s3storage.transient.bucket=${env:NUXEO_S3STORAGE_TRANSIENT_BUCKET:}
+nuxeo.s3storage.transient.roleArn=${env:NUXEO_S3STORAGE_TRANSIENT_ROLEARN:}
+nuxeo.s3storage.transient.targetMaxSizeMB=${env:NUXEO_S3STORAGE_TRANSIENT_TARGETMAXSIZEMB:100}
+nuxeo.s3storage.transient.ttl=${env:NUXEO_S3STORAGE_TRANSIENT_TTL:4320}
+nuxeo.s3storage.transient.ttl2=${env:NUXEO_S3STORAGE_TRANSIENT_TTL2:10}
+
+# usually taken from system environement
+nuxeo.s3storage.transient.awsid=${env:NUXEO_S3STORAGE_TRANSIENT_AWSID:}
+nuxeo.s3storage.transient.awssecret=${env:NUXEO_S3STORAGE_TRANSIENT_AWSSECRET:}
+nuxeo.s3storage.transient.awstoken=${env:NUXEO_S3STORAGE_TRANSIENT_AWSTOKEN:}
+nuxeo.s3storage.transient.region=${env:NUXEO_S3STORAGE_TRANSIENT_REGION:}
+
+# optional
+nuxeo.s3storage.transient.endpoint=${env:NUXEO_S3STORAGE_TRANSIENT_ENDPOINT:}
+nuxeo.s3storage.transient.expiration=${env:NUXEO_S3STORAGE_TRANSIENT_EXPIRATION:}
+nuxeo.s3storage.transient.pathstyleaccess=${env:NUXEO_S3STORAGE_TRANSIENT_PATHSTYLEACCESS:false}
+nuxeo.s3storage.transient.accelerateMode=${env:NUXEO_S3STORAGE_TRANSIENT_ACCELERATEMODE:false}
+nuxeo.s3storage.transient.bucket_prefix=${env:NUXEO_S3STORAGE_TRANSIENT_BUCKET_PREFIX:}
+nuxeo.s3storage.transient.crypt.serverside=${env:NUXEO_S3STORAGE_TRANSIENT_CRYPT_SERVERSIDE:false}
+
+
+## postgresql confs mapping
+
+nuxeo.db.type=${env:NUXEO_DB_TYPE:postgresql}
+nuxeo.db.host=${env:NUXEO_DB_HOST:localhost}
+nuxeo.db.port=${env:NUXEO_DB_PORT:5432}
+nuxeo.db.name=${env:NUXEO_DB_NAME:nuxeo}
+nuxeo.db.user=${env:NUXEO_DB_USER:nuxeo}
+nuxeo.db.password=${env:NUXEO_DB_PASSWORD:password}
+nuxeo.db.jdbc.url=${env:NUXEO_DB_JDBC_URL:jdbc:postgresql://localhost:5432/nuxeo}
```

Comment:
Let's just expose the url and the login/pwd

Reply:
Individual values are needed in the XA XML configuration i think.
The url is the one usually resolved thanks to other variables (original value: nuxeo.db.jdbc.url=jdbc:postgresql://${nuxeo.db.host}:${nuxeo.db.port}/${nuxeo.db.name}) so i'll remove this one if confirmed that this format will be the same here, to avoid repetition.

Reply:
assumed the original value would match the need: removed this last line to avoid repetition